### PR TITLE
Add checklist to PR

### PR DESCRIPTION
**Pull request by YohannParis**
# Description

Add a simple checlist to a PR, which, when all items are checked, get the PR approved

### CODE REVIEW

**Comment by YohannParis** [comment]
Edited Test comment

**Comment by YohannParis** [comment]
> Are these checklist points meant for the reviewers on the PR? Or the people/person that raised the PR?
> 
> If this is meant for the latter, I feel like we will get better mileage by leading with good examples of what PR descriptions look like - because having a non-descriptive PR the reviewers still wouldn't know what to look for despite everything being check marked.

You make an excellent point.
The problem is noboy uses the templates for PRs, we tried, but people do not edit them. The thinking was to get an extra step for people to think about before merging.

File: .github/workflows/checklist.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+            // Create a new comment if the bot hasn't commented yet
+            if (!botComment) {
+              console.log("No bot comment found. Creating a new comment.");
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.payload.number,
+                body: "**Please review the checklist** \n\n- [ ] _manual test_ and/or _scenario test_ updated, if not sure ask #askem-testing\n- [ ] _documentation_ updated\n- [ ] _breaking change_ notify #askem-engineering",
+              })
```
Comment (dgauldie) [inline .github/workflows/checklist.yml:50]: server error messages use `messages.properties`

Comment (YohannParis) [reply .github/workflows/checklist.yml:50]: Done

**Review by dgauldie — APPROVED** [review]

**Pull request merged**